kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -539,4 +522,70 @@ def _document(cls):
             n=cls._doc_array_n)
 
 
-__all__ = ['ScatteringBase1D']
+class TimeFrequencyScatteringBase(ScatteringBase1D):
+    def __init__(self, *, J, J_fr, shape, Q, T=None, oversampling=0,
```

Comment:
can you make this `*args`

Reply:
https://peps.python.org/pep-3102/
keyword-only arguments